lib/jsonschema.lua

```diff
@@ -136,70 +134,61 @@ end
 
 -- load doesn't like at all empty string, but sometimes it is easier to add
 -- some in the chunk buffer
-local function yield_chunk(chunk)
+local code_table = {}
```

Comment:
In general, we do not recommend using global tables to cache data. It is very dangerous for multiple objects to share a global table.

We should put the code_table object inside `self` object.

Reply:
https://github.com/api7/jsonschema/pull/13/files#diff-c2f04d3e8610e8986b421855dca6cc54R148

change to:

```lua
self:top_code(indent)
```

Reply:
@membphis I have refactored accordingly and passed the local variable _code_table to the generate function. `self:insert_code(indent)` is a bit tricky as it involves the child processes as well and `insert_code` is only available for the root element.

With the current refactoring, it should resolve the issue as we are passing the self. variable to the downwards functions.